src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue20438.cs

```diff
@@ -0,0 +1,32 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+
+public class Issue20438 : _IssuesUITest
+{
+	public Issue20438(TestDevice testDevice) : base(testDevice)
+	{
+	}
+	public override string Issue => "Add DateOnly and TimeOnly converters to DatePicker and TimePicker";
+
+	[Test]
+	[Category(UITestCategories.DatePicker)]
+	public void DatePickerConverters()
+	{
+		Assert.That(App.WaitForElement("LabelDateOnly").GetText(), Is.EqualTo("DateOnly Value: 2025-07-28"));
```

Comment:
not sure this test is resistant to localisation changes, and probably won't pass on all machines